drivers/include/hdc1000.h

```diff
@@ -9,126 +10,137 @@
 /**
  * @defgroup    drivers_hdc1000 HDC1000 Humidity and Temperature Sensor
  * @ingroup     drivers_sensors
- * @brief       Driver for the Texas Instruments HDC1000
- *              Humidity and Temperature Sensor.
- *              The driver will initialize the sensor for best
- *              resolution (14 bit). Currently the driver doesn't use the heater.
- *              Temperature and humidity are acquired in sequence.
- *              The sensor is always in sleep mode. The measurement must
- *              be started by a write access to the address 0x00
- *              (HDC1000_TEMPERATURE). After completing the measurement
- *              the sensor will return to sleep mode. Typical
- *              Conversion Time by 14 bit resolution is 6.50ms
- *              for humidity and 6.35ms for temperature.
- *              HDC1000_CONVERSION_TIME is twice as large to prevent
- *              the problems with timer resolution.
+ * @brief       Driver for the TI HDC1000 Humidity and Temperature Sensor
+ *
+ * The driver will initialize the sensor for best resolution (14 bit). Currently
+ * the driver doesn't use the heater. Temperature and humidity are acquired in
+ * sequence. The sensor is always in sleep mode.
+ *
+ * The temperature and humidity values can either be acquired using the
+ * simplified `hdc1000_read()` function, or the conversion can be triggered
+ * manually using the `hdc1000_trigger_conversion()` and `hdc1000_get_results()`
+ * functions sequentially. If using the second method, on must wait at least
+ * `HDC1000_CONVERSION_TIME` between triggering the conversion and reading the
+ * results.
+ *
+ * @note        The driver does currently not support using the devices heating
+ *              unit.
  *
  * @{
  *
  * @file
- * @brief       Interface definition for the HDC1000 sensor driver.
+ * @brief       Interface definition for the HDC1000 sensor driver
  *
  * @author      Johann Fischer <j.fischer@phytec.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
  */
 
 #ifndef HDC1000_H
 #define HDC1000_H
 
 #include <stdint.h>
-#include <stdbool.h>
+
 #include "periph/i2c.h"
+#include "hdc1000_regs.h"
 
 #ifdef __cplusplus
 extern "C"
 {
 #endif
 
+/**
+ * @brief   Default I2C bus address of HDC1000 devices
+ */
 #ifndef HDC1000_I2C_ADDRESS
-#define HDC1000_I2C_ADDRESS           0x43 /**< Default Device Address */
+#define HDC1000_I2C_ADDRESS           (0x43)
 #endif
 
+/**
+ * @brief   Typical conversion time needed to acquire new values [in us]
+ *
+ * @note    This time value is chosen twice as large as needed for 14-bit
+ *          conversions (worst case) to allow for timer imprecision
+ */
 #ifndef HDC1000_CONVERSION_TIME
-#define HDC1000_CONVERSION_TIME       26000 /**< Default Conversion Time */
+#define HDC1000_CONVERSION_TIME       (26000)
```

Comment:
Ok, "for each value" makes sense but why times two? Because 14 Bit needs two bytes and that is a "value"? Well, I don't want to block because of that value. The test should get priority (adaption or deleting)